### DISCUSSION TITLE
Not loading metadata for MODEL_FILENAME as it lacks a proper header (path='PATH_TO_MODEL')

### DISCUSSION THREAD

**Discussion by Jonseed** [mcmonkeyprojects/SwarmUI#404] [Q&A]
The debug logs show this for many of my models, and I can't use them in SwarmUI. But I can use them fine directly in ComfyUI. What does it mean? This shows up on my GGUF models.

On the GGUF Flux models it also says "did not match any of 64 options" and "has special format 'gguf'."

If I try to use one of these models I get the error:
`_pickle.UnpicklingError: invalid load key, '\x03'.`

I have many other models that load fine.

**Comment by mcmonkey4eva** [comment]
Post on Discord in the help-forum channel, or as a User Support post on the Issues tab here.
